src/Pages/Main/Modal/Calendar.js

```diff
@@ -0,0 +1,25 @@
+import styled from 'styled-components/macro';
+
+import React from 'react';
```

Comment:
```suggestion
import React from 'react';
import styled from 'styled-components/macro';
```
Import 순서 수정해주세요! 일반적인 convention을 따르는 이유도 있지만 순서만 잘 지켜주셔도 가독성이 좋아집니다. 아래 순서 참고해주세요.

- React → Library(Package) → Component → 변수 / 이미지 → css 파일(scss 파일)

Reply:
네 다른 js 파일도 수정했습니다